ansible_collections/arista/cvp/plugins/module_utils/tag_tools.py

```diff
@@ -136,6 +138,13 @@ def tasker(self, tags: list, mode: string, auto_create: bool = True):
                         self.__cv_client.api.tag_config(element_type, workspace_id,
                                                         tag_name, tag_val)
                     if mode == 'delete':
+                        # unassign tags first
+                        self.__cv_client.api.tag_assignment_config(element_type,
+                                                                   workspace_id,
+                                                                   tag_name,
+                                                                   tag_val,
+                                                                   device_id,
+                                                                   "")
```

Comment:
Fixed the code but we are still getting positive output (success: true) for both delete and assign (auto_create=False) from API, altho on the lab I don't see tags anymore (manually).
API output shows workspace got created successfully. 
Now, since there is no way to check if above cases worked or not using assertion statement do you want me to remove the negative testcases?

Example Output:
ok: [CloudVision] => 
  msg:
    changed: true
    failed: false
    success: true
    tags_manager:
      changed: true
      diff: {}
      success: true
      tags_manager_count: 1
      tags_manager_list:
      - tag_AnsibleWorkspaceFBM
      taskIds: []
    taskIds: []

Reply:
Made similar changes for interface

Reply:
Keeping Negative test-cases commented till further instructions.